pipenv/utils/virtualenv.py

```diff
@@ -237,6 +238,22 @@ def abort(msg=""):
     # Find out which python is desired.
     if not python:
         python = project.required_python_version
+        if python:
+            range_pattern = r"^[<>]=?|!="
+            if re.search(range_pattern, python):
+                click.echo(
```

Comment:
We have been migrating away from click.echo to use the rich console -- you can find other examples in the code of how to print the equivalent thing using that.

Reply:
@matteius Thanks for guidance. I'll check and modify it to the convention.